source/less/blocks/advantages.less

```
@@ -26,14 +28,89 @@
 }
 
 .advantages__item {
+  min-height: 60px;
+  box-sizing: border-box;
+  position: relative;
+  padding-top: 2px;
+  padding-bottom: 4px;
   padding-left: 80px;
+  margin-bottom: 20px;
+}
+
+.advantages__item::before {
+  content: "";
+  position: absolute;
+  width: 60px;
+  height: 60px;
+  background-color: @color-accent;
+  top: 0;
+  left: 0;
+}
+
+.advantages__item::after {
+  content: "";
+  position: absolute;
+}
+
+.advantages__item:nth-child(1)::after {
+  width: 30px;
+  height: 32px;
+  top: 14px;
+  left: 15px;
+  content: url("../../img/index/svg/icon-leaf.svg");
```

```suggestion
  content: url("../img/index/svg/icon-leaf.svg");
```
Был лишний «выход» из папки. Тебе нужно опираться не на текущее расположение файла со стилями, а итогового, который будет подключаться на странице после сборки (style.min.css)